test-cypress/integration/graphql/graphql-playground-query.spec.js

```diff
@@ -0,0 +1,40 @@
+import {RepositoriesStubs} from "../../stubs/repositories/repositories-stubs";
+import {GraphqlPlaygroundSteps} from "../../steps/graphql/graphql-playground-steps";
+import {GraphqlStubs} from "../../stubs/graphql/graphql-stubs";
+import {ToasterSteps} from "../../steps/toaster-steps";
+import {MonitoringStub} from "../../stubs/monitoring-stub";
+
+describe('GrpahQL query', () => {
+
+    let repositoryId;
+
+    beforeEach(() => {
+        repositoryId = 'abort-graphql-query' + Date.now();
+        cy.createRepository({id: repositoryId});
+        cy.presetRepository(repositoryId);
+        cy.importServerFile(repositoryId, 'swapi-dataset.ttl');
+        cy.uploadGraphqlSchema(repositoryId, 'graphql/soml/swapi-schema.yaml', 'swapi');
+        cy.uploadGraphqlSchema(repositoryId, 'graphql/soml/swapi-schema-planets.yaml', 'swapi-planets');
+        RepositoriesStubs.spyGetRepositories();
+    });
+
+    afterEach(() => {
+        cy.deleteRepository(repositoryId);
+    });
+
+    it('Should abort the query and notify about it', () => {
+        // Given: I visit a page containing a GraphQL playground element.
+        GraphqlPlaygroundSteps.visit();
+        GraphqlStubs.delayGraphQLQuery(repositoryId);
+        MonitoringStub.interceptDeleteSparqlQuery(repositoryId);
+
+        // When: I cancel a long-running query.
+        GraphqlPlaygroundSteps.executeQuery();
```

Comment:
I can't run this test just now, but I wonder, what is the query you are executing here?

Reply:
For the test, the query does not matter. Whether it is correct and returns successfully or returns an error, the goal of the test is to check if a running query is aborted.